### PR TITLE
#160 Prevent rerender of background Image on tab change

### PR DESCRIPTION
**Pull request by brooksbecton**
fixes: #160 

**Note**: Could I get the `hacktoberfest-accepted` label on this PR if everything looks ok? 

I edited the example to show that the background image will still re render if the source uri changes by showing a different background image on refresh
Large GIF (378x758)

### CODE REVIEW

File: src/StickyParallaxHeader.js [hunk 1]
```diff
@@ -29,6 +29,35 @@ import { getSafelyScrollNode, setRef } from './utils';
 const { divide, Value, createAnimatedComponent, event, timing, ValueXY } = Animated;
 const AnimatedScrollView = createAnimatedComponent(ScrollView);
 
+const HeaderBackgroundImage = (props) => {
```
Comment (maciejBudzinskiNG) [inline src/StickyParallaxHeader.js:32]: The component looks good, but I would like you to move it (including styles) to a separate file under the src/components/HeaderBackgroundImage directory.

File: src/StickyParallaxHeader.js [hunk 2]
```diff
@@ -29,6 +29,35 @@ import { getSafelyScrollNode, setRef } from './utils';
...
+};
+
+HeaderBackgroundImage.propTypes = {
+  background: node,
+  backgroundHeight: number,
+  backgroundImage: Image.propTypes.source,
+};
+
+const headerImagesAreEqual = (prevProps, props) =>
+  prevProps.backgroundImage.uri === props.backgroundImage.uri;
```
Comment (maciejBudzinskiNG) [inline src/StickyParallaxHeader.js:57]: Don't we need to add conditions for other props as well?
If `backgroundHeight` or `background` prop changes, the component still needs to rerender.

**Review by maciejBudzinskiNG — CHANGES_REQUESTED** [review]

**Review by maciejBudzinskiNG — CHANGES_REQUESTED** [review]
Thank you for your work on this issue. Check my comments 🙂

**Comment by brooksbecton** [comment]
I've made these changes.

**Comment by Ciberusps** [comment]
🙃

**Review by IdaszakDaniel — APPROVED** [review]

**Pull request merged**